doc/manipulating_images/manipulating_images.rst

```diff
@@ -59,50 +59,59 @@ follows::
 
 |
 
+.. seealso::
 
-For a list of all the data fetching functions in nilearn, see :ref:`datasets_ref`.
+    For a list of all the data fetching functions in nilearn, see
+    :ref:`datasets_ref`.
 
-Besides convenient downloading of openly accessible reference datasets
-including important meta-data (e.g., stimulus characteristics and
-participant information for confound removal), the fetching functions
-perform data downloads only once and return the locally saved data upon
-any later function calls.
-The locally stored data can be found in one of the
-following directories (in order of priority, if present):
+.. topic:: **nilearn_data: Where is the downloaded data stored?**
 
-  * default system paths used by third party software that may already
-    provide the data (e.g., the Harvard-Oxford atlas
-    is provided by the FSL software suite)
-  * the folder specified by `data_dir` parameter in the fetching function
-  * the global environment variable `NILEARN_SHARED_DATA`
-  * the user environment variable `NILEARN_DATA`
-  * the `nilearn_data` folder in the user home folder
+    The fetching functions download the reference datasets to the disk.
+    They save it locally for future use, in one of the
+    following directories (in order of priority, if present):
 
-Two different environment variables are provided to distinguish a global dataset
-repository that may be read-only at the user-level.
-Note that you can copy that folder to another user's computers to avoid
-the initial dataset download on the first fetching call.
+     * the folder specified by `data_dir` parameter in the fetching function
+     * the global environment variable `NILEARN_SHARED_DATA`
+     * the user environment variable `NILEARN_DATA`
+     * the `nilearn_data` folder in the user home folder
 
+    The two different environment variables (NILEARN_SHARED_DATA and
+    NILEARN_DATA) are provided for multi-user systems, to distinguish a
+    global dataset repository that may be read-only at the user-level.
```

Comment:
> global may not be clear to everybody here

OK but:

- This is not a change introduced by this PR
- This paragraph is for advanced users

So, skipping for now.